scripts/New-VersionConstantsFile.ps1

```diff
@@ -25,7 +25,7 @@ namespace $namespace
         public const string Prerelease = "$versionSuffix";
         public const string AssemblyVersion = "$versionPrefix";
         public const string FileVersion = AssemblyVersion + ".0";
-        public const string Version = AssemblyVersion + Prerelease;
+        public const string SemanticVersion = AssemblyVersion + "-" + Prerelease;
```

Comment:
>SemanticVersion [](start = 28, length = 15)

Is this going to work when we don't actually have any prerelease data? Looks like we'll end up sticking a hyphen at the end of our final release.

Reply:
Nope, that's a bug. So I won't be able to get away with `SemanticVersion` being a compile-time constant, so I'll expose it as a computed property in `SarifUtilities`. Pity.

As I mentioned elsewhere in my comments, the `Version` property as defined wasn't used anywhere, so I'll just stop generating it.

---
In reply to: [221750223](https://github.com/Microsoft/sarif-sdk/pull/1105#discussion_r221750223) [](ancestors = 221750223)

Reply:
Fixed.

---
In reply to: [221759286](https://github.com/Microsoft/sarif-sdk/pull/1105#discussion_r221759286) [](ancestors = 221759286,221750223)